src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: string,
```

Comment:
You have created type for `shape (Shape)`  and you use if at 6 line, but you didn't use `Color `for `color `in `Figure `interface.